planterbox/__init__.py

```diff
@@ -285,8 +285,12 @@ class Planterbox(Plugin):
     def register(self):
         super(Planterbox, self).register()
         if 'start_time' not in self.config._mvd:
-            start_time = datetime.now().isoformat()
+            start_date = datetime.now().strftime("%Y-%m-%d")
+            start_time = datetime.now().strftime("%H_%M_%S")
```

Comment:
I recommend taking `datetime.now()` once and then `strftime`ing the resulting value twice. It mostly won't matter, but mostly means "confusing results if run at midnight".